Protobuild script targets .NET 3.5 on Windows GL build

In the MG windows GL build on a windows 8 box with only .NET 4, the lidgren build fails:

```
             [exec]   C:\Windows\Microsoft.NET\Framework\v4.0.30319\Microsoft.Common.targets(983,5): error MSB3645: .NET Framework v3.5 Service Pack 1 was not found. In order to target ".NETFramework,Version=v3.5", .NET Framework v3.5 Service Pack 1 or later must be installed. 
```

[MonoGame\ThirdParty\Lidgren.Network\Lidgren.Network.Windows.csproj]

Looks like the protobuild script is generating targets for v3.5 instead of 4?

--- a/ThirdParty/Lidgren.Network/Lidgren.Network.Windows.csproj
+++ b/ThirdParty/Lidgren.Network/Lidgren.Network.Windows.csproj
@@ -11,20 +11,7 @@
     <RootNamespace>Lidgren.Network</RootNamespace>
     <AssemblyName>Lidgren.Network</AssemblyName>
     <FileAlignment>512</FileAlignment>
-    <SccProjectName>
-    </SccProjectName>
-    <SccLocalPath>
-    </SccLocalPath>
-    <SccAuxPath>
-    </SccAuxPath>
-    <SccProvider>
-    </SccProvider>
-    <FileUpgradeFlags>
-    </FileUpgradeFlags>
-    <UpgradeBackupLocation>
-    </UpgradeBackupLocation>
-    <OldToolsVersion>3.5</OldToolsVersion>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>

